commit 026b95145e415dab2a0dd82deb2cb852a26df157 (refs/changes/71/4871/2)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-09-25T11:51:39-04:00 (1 year ago)
    
    OS-7262 sysinfo generates invalid JSON due to HVM API
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/src/sysinfo b/src/sysinfo
index f1dbf980..8f1a3f72 100644
--- a/src/sysinfo
+++ b/src/sysinfo
@@ -211,6 +211,7 @@ function get_bhyve_capability()
 {
     Bhyve_Capable="false"
     Bhyve_Max_Vcpus="0"
+    HVM_API="false"
 
     if [[ -x /usr/lib/brand/bhyve/bhhwcompat ]]; then
         if /usr/lib/brand/bhyve/bhhwcompat; then
